app/Models/Employee.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Employee extends Model
+{
+    use HasFactory;
+
+    // protected $fillable = [
+    //     'employee_code',
+    //     'employee_fullname',
+    //     'phone_number',
+    //     'address',
+    //     'role'
+    // ];
```

Comment:
ga dipake ta?
```suggestion
protected $guarded = [];
```
bisa pake yg diatas klo attributenya gaada yg di protect, jadi gaperlu ngisi attribute fillable

Reply:
@Zaki140502  iki ga diubah ta?